packages/local-environment/src/nomadenv.ts

```diff
@@ -141,6 +145,23 @@ export class NomadEnv {
     return false;
   }
 
+  get forkUrl(): string | undefined {
+    if (process.env.ALCHEMY_API_KEY)
+      return `https://eth-mainnet.g.alchemy.com/v2/${process.env.ALCHEMY_API_KEY}`;
+  }
```

Comment:
What if Alchemy key is provided, but we work with not a mainnet? Also I don't think it has anything to do with a NomadEnv in general. It is purely a Network layer thing. In particular HardhatNetwork

Reply:
Ill replace with the fork url in .env

Reply:
But also it is probably not the place for it in the first place. What if we want to have Tom a forked net and Jerry hardhat from scratch or something similar.

Reply:
Okay then, removing